console/console-init/ui/src/modules/iot-device/DeviceListPage.tsx

```diff
@@ -44,8 +47,39 @@ export default function DeviceListPage() {
     getInitialFilter()
   );
 
+  const [isAlertVisible, setIsAlertVisible] = useState<boolean>(false);
```

Comment:
Instead of creating 4 state variables we can create a single object for alert variables like 
`const [alert,setAlert] = useState<{isVisible:boolean;variant:AlertVariant;title:string;description}> `

Reply:
done